starboard/shared/win32/posix_emu/include/sys/stat.h

```diff
@@ -16,13 +16,21 @@
 
 #include <../ucrt/sys/stat.h>
 
+#include <direct.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #define S_ISLNK(mode) 0  // Windows doesn't support symbolic links
 #define S_ISDIR(mode) (((mode) & _S_IFMT) == (_S_IFDIR))
 
+typedef int mode_t;
+
+int sb_mkdir(const char* path, mode_t mode);
+#undef mkdir
+#define mkdir sb_mkdir
```

Comment:
@gbournou I thought we were planning to use wrapping instead of macros?

Reply:
Windows has an existing mkdir implementation, so this is similar to https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/shared/win32/posix_emu/include/malloc.h;l=26

Reply:
Got it, thanks for the clarification